cg/server/dto/sequencing_metrics/sequencing_metrics_request.py

```diff
@@ -1,6 +1,6 @@
 """DTO for the sequencing metrics."""
 
-from pydantic import BaseModel
+from pydantic import BaseModel, Field
 
 
 class SequencingMetricsRequest(BaseModel):
```

Comment:
if possible change the `flow_cell_name` attribute  to `flow_cell_id`

Reply:
Not possible without a change on the other side.